README.md

```diff
@@ -15,6 +15,13 @@
 - `yarn update-server` ausführen, damit der server im Verzeichnis `server` verfügbar wird (dieser Schritt muss zum updaten des servers wiederholt werden)
 - GUI per `yarn start` starten
 
+### Für Windows-User
```

Comment:
Ich sehe keinen Grund, einen eigenen Abschnitt zu erstellen, wenn die Punkte sich nur minimal unterscheiden, das macht das Readme unnötig lang. Stattdessen kannst du in den Punkten die OS-Besonderheiten anmerken, wenn es nach den anderen Änderungen überhaupt noch welche gibt.

Reply:
Sollte ich das in den Unterpunkt mögliche Probleme ganz unten einfach hinein schieben?